exams/utils.py

```diff
@@ -41,8 +41,12 @@ def authorize_for_exam(mmtrack, course_run):
 
         # if user passed the course and currently not authorization for that run then give
         # her authorizations.
+        try:
+            passed = mmtrack.has_passed_course(course_run.edx_course_key)
+        except:  # pylint: disable=bare-except
+            return
```

Comment:
Is there a reason we don't log the exception here? I worry about this bare exception hiding quiet failures which will make it difficult to debug.